spec/taxonomy_sidebar_spec.rb

```diff
@@ -41,6 +41,54 @@
           )
       end
 
+      it 'only shows related links for the first 2 taxons with related  content' do
```

Comment:
Really minor suggestions:

- I think this test would be clearer if it was swapped with the next one, so that you have a test of related content and then a test of _limited_ related content.
- Typo: double space in `related  content`.